styles/views/challenge.styl

```
@@ -1,10 +1,22 @@
 .page-challenge
 
-    .button-hint
-        button-color(color-editor-bg)
+    .inner-header 
+        display: flex
+        justify-content: space-between;
```

Looks like the rest of the file doesn't have any colons or semicolons - stay consistent with the file.

```suggestion
        justify-content space-between
```